index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title> Leslie Portfolio project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body class="body">
+    <header class="header">
+      <nav class="navigate">
+        <a href=""><img src="images/John Doe.png" alt="" /></a>
```

Comment:
- [ ] According to the project template. there is a space around the logo and the nav hamburger, i.e space in the top, right, bottom, and left. As seen below: 👇 

![head1](https://user-images.githubusercontent.com/78634841/204928408-54130da1-550e-40bd-8d9b-32633b0f06fb.png)

compare to what we have in your project: 👇 

![cor](https://user-images.githubusercontent.com/78634841/204928736-689b02c5-cff5-4939-b131-d714209e63b1.png)

- Kindly check the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=DSRLSZBE6X8joMdm-0) for proper styling of the navbar header